webpack/components/extensions/HostDetails/Tabs/TraceTab.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useCallback } from 'react';
+import { Skeleton, Button } from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { TableVariant, Thead, Tbody, Tr, Th, Td } from '@patternfly/react-table';
+import { useSelector } from 'react-redux';
+import { selectAPIResponse } from 'foremanReact/redux/API/APISelectors';
+import TableWrapper from '../../../../components/Table/TableWrapper';
+import getHostTraces from './HostTracesActions';
+import { selectHostTracesStatus } from './HostTracesSelectors';
+
+const TracesTab = () => {
+  const [searchQuery, updateSearchQuery] = useState('');
+  const hostDetails = useSelector(state => selectAPIResponse(state, 'HOST_DETAILS'));
+  const { id: hostId } = hostDetails;
+  const emptyContentTitle = __('This host currently does not have traces.');
+  const emptyContentBody = __("Add filters using the 'Add filter' button above."); // needs link
+  const emptySearchTitle = __('No matching traces found');
+  const emptySearchBody = __('Try changing your search settings.');
+  const actionButtons = <Button variant="danger" isDisabled> Restart app </Button>;
+  const fetchItems = useCallback(
+    params =>
+      (hostId ? getHostTraces(hostId, params) : null),
+    [hostId],
+  );
+  const response = useSelector(state => selectAPIResponse(state, 'HOST_TRACES'));
+  const { results, ...meta } = response;
+  const status = useSelector(state => selectHostTracesStatus(state));
+  if (!hostId) return <Skeleton />;
+  /* eslint-disable max-len */
+  return (
+    <>
```

Comment:
The table needs some styling added so it has some left and right margin, and the text blurb at the top needs spacing to look more like the mockup.  To do that,

1. instead of returning a `<>` Fragment here, let's make it a `<div>` with an id:
```suggestion
    <div id="traces-tab">
```
2. Make a `TracesTab.scss` in the same folder and put the CSS there:
```css
#traces-tab {
  margin: 0 20px;
  h3 {
    margin-bottom: 2em;
  }
}
```
3. You'll also have to 
```js
import './TracesTab.scss';
```
at the top of this file.

Reply:
Fixed.